app/src/main/java/edu/outside2154/gamesense/model/Player.kt

```diff
@@ -12,9 +12,9 @@ const val PLAYER_BASE_ATTACK = 100.0
 const val PLAYER_CRIT_MULT = 2.0
 
 interface Player : Serializable {
-    val regenStat: Stat
-    val atkStat: Stat
-    val intStat: Stat
+    var regenStat: Stat
```

Comment:
It's probably best to keep them as `val`, due to the goal progress bug I mentioned above. For now it's fine, but let's add a `TODO`.